src/main/scala/config/MetricsSettings.scala

```diff
@@ -22,7 +22,8 @@ object MetricsSettings {
       CloudWatch(
         config.getBoolean("metrics.cloudwatch.enabled"),
         config.getString("metrics.cloudwatch.name"),
-        config.getInt("metrics.cloudwatch.interval_minutes")
+        config.getInt("metrics.cloudwatch.interval_minutes"),
+        config.getString("metrics.cloudwatch.metric_filter")
```

Comment:
Perhaps this would be nicer as an optional string, so that you can clean up the `length == 0` comparisons later

Reply:
i dont think config has an getoptionalstring method.